<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>699</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lisa Backup - woof!</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="tasksWrap" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="editorWrap" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="pageEdit">
           <layout class="QVBoxLayout" name="verticalLayout_17">
            <item>
             <widget class="QLabel" name="labelTaskHeading">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">
	font-weight: bold;
font-size: 15px;

</string>
              </property>
              <property name="text">
               <string>XYZ</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBoxSourceList">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>Sources</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QSplitter" name="splitterSources">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="childrenCollapsible">
                  <bool>false</bool>
                 </property>
                 <widget class="QWidget" name="layoutWidget">
                  <layout class="QVBoxLayout" name="verticalLayout_6">
                   <item>
                    <widget class="QListView" name="sourcesListView">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="font">
                      <font>
                       <bold>false</bold>
                      </font>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QWidget" name="widget" native="true">
                     <property name="font">
                      <font>
                       <bold>false</bold>
                      </font>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout_2">
                      <property name="spacing">
                       <number>13</number>
                      </property>
                      <property name="leftMargin">
                       <number>0</number>
                      </property>
                      <property name="rightMargin">
                       <number>0</number>
                      </property>
                      <item>
                       <widget class="QToolButton" name="toolButtonAddSource">
                        <property name="minimumSize">
                         <size>
                          <width>32</width>
                          <height>32</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>Add</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QToolButton" name="toolButtonRemoveSource">
                        <property name="minimumSize">
                         <size>
                          <width>32</width>
                          <height>32</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>Remove</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <spacer name="horizontalSpacer_2">
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>196</width>
                          <height>20</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                      <item>
                       <widget class="QToolButton" name="toolButtonSourceUp">
                        <property name="minimumSize">
                         <size>
                          <width>32</width>
                          <height>32</height>
                         </size>
                        </property>
                        <property name="toolTip">
                         <string>Change backup order for selected source</string>
                        </property>
                        <property name="text">
                         <string/>
                        </property>
                        <property name="arrowType">
                         <enum>Qt::NoArrow</enum>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QToolButton" name="toolButtonSourceDown">
                        <property name="minimumSize">
                         <size>
                          <width>32</width>
                          <height>32</height>
                         </size>
                        </property>
                        <property name="toolTip">
                         <string>Change backup order for selected source</string>
                        </property>
                        <property name="text">
                         <string/>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBoxDestination">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="title">
               <string>Destination</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_15">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_15">
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_Destination">
                     <property name="spacing">
                      <number>6</number>
                     </property>
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_6">
                       <item>
                        <widget class="QLineEdit" name="lineEditDestinationPath">
                         <property name="sizePolicy">
                          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                           <horstretch>0</horstretch>
                           <verstretch>0</verstretch>
                          </sizepolicy>
                         </property>
                         <property name="text">
                          <string/>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QPushButton" name="pushButtonChooseDestinationSubdir">
                         <property name="sizePolicy">
                          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                           <horstretch>0</horstretch>
                           <verstretch>0</verstretch>
                          </sizepolicy>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>40</width>
                           <height>16777215</height>
                          </size>
                         </property>
                         <property name="text">
                          <string>...</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QToolButton" name="toolButtonOpenDirExternal">
                         <property name="text">
                          <string>...</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_DestinationWarning">
                       <property name="font">
                        <font>
                         <pointsize>9</pointsize>
                         <kerning>true</kerning>
                        </font>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">color:orange;</string>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayoutSaveRow">
              <item>
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QToolButton" name="toolButtonSaveTask">
                <property name="minimumSize">
                 <size>
                  <width>32</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="text">
                 <string>Save</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBoxTriggering">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>Triggering</string>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_14">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_Triggering">
                     <item>
                      <widget class="QToolButton" name="toolButtonRefreshTriggering">
                       <property name="minimumSize">
                        <size>
                         <width>32</width>
                         <height>32</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>Refresh</string>
                       </property>
                       <property name="icon">
                        <iconset>
                         <normalon>:/custom-icons/refresh.svg</normalon>
                        </iconset>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="pageNone">
           <layout class="QVBoxLayout" name="verticalLayout_19">
            <property name="topMargin">
             <number>9</number>
            </property>
            <property name="bottomMargin">
             <number>9</number>
            </property>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_20">
               <property name="topMargin">
                <number>9</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <item>
                  <widget class="QLabel" name="labelNoTaskSelected">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>&lt;a href=&quot;create_task&quot;&gt;Create&lt;/a&gt; a new task or select one form the list on the left...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Tasks</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionContents">
   <property name="text">
    <string>Contents</string>
   </property>
  </action>
  <action name="action_New">
   <property name="icon">
    <iconset>
     <normalon>:/custom-icons/file-plus.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_ManageTasks">
   <property name="text">
    <string>&amp;Manage...</string>
   </property>
   <property name="iconText">
    <string>Manage tasks</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normalon>:/custom-icons/save.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset>
     <normalon>:/custom-icons/settings.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Se&amp;ttings...</string>
   </property>
  </action>
  <action name="actionShow_trigger_monitor">
   <property name="text">
    <string>Trigger monitor...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
